.github/workflows/lint.yaml

```diff
@@ -1,11 +1,5 @@
-name: Lint
-
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-
+name: PHP Lint CI
+on: [push, pull_request]
```

Comment:
Does this mean a branch is linted twice: once on push and again when the PR is created?

Reply:
Yeah, I need to remove `pull_request`. Good catch.